docs/griptape-framework/engines/image-query-engines.md

```diff
@@ -5,16 +5,21 @@ The [Image Query Engine](../../reference/griptape/engines/image_query/image_quer
 ```python
 from griptape.drivers import OpenAiVisionImageQueryDriver
 from griptape.engines import ImageQueryEngine
-from griptape.loaders import ImageLoader 
+from griptape.loaders import ImageLoader
 
 driver = OpenAiVisionImageQueryDriver()
 
 engine = ImageQueryEngine(
     image_query_driver=driver,
+    max_tokens=100
```

Comment:
Whoops, just noticed this. The `max_tokens` parameter should be set when we initialize the driver, not the engine.

Reply:
ah, good catch! Just fixed.